manifests/e/evernote/yinxiang/7.0.6.4762/evernote.yinxiang.installer.yaml

```diff
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.0.7 $debug=MDSU.5-1-22000-282
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: evernote.yinxiang
+PackageVersion: 7.0.6.4762
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /quiet
+  SilentWithProgress: /passive
+  Custom: /norestart
+FileExtensions:
+- enex
+Installers:
+- Architecture: x86
+  ProductCode: '{7D04FD44-8AEB-11EC-A605-1402EC34E358}'
+  InstallerUrl: https://cdn-dl.yinxiang.com/YXWin6/public/Evernote_7.0.6.4762.exe
+  InstallerSha256: E9270E462512D4D0536EEC9A84FC44767F0B481EBAEB72AB36D34CC0F613A2AF
```

Comment:
```suggestion
  InstallerSha256: E9270E462512D4D0536EEC9A84FC44767F0B481EBAEB72AB36D34CC0F613A2AF
  AppsAndFeaturesEntries:
  - Publisher: Beijing Yinxiang Biji Technologies Co., Ltd.
    DisplayName: Yinxiang Biji v. 7.0.6
    DislayVersion: 7.0.6.4762
    InstallerType: msi
    ProductCode: '{7D04FD44-8AEB-11EC-A605-1402EC34E358}'
```

Reply:
I tested and didn't work either.  Because this package's ProductCode is different from previous version.

Reply:
You need winget-cli v1.3 to get the effect of AppsAndFeaturesEntries

Reply:
I had tested it on v1.3.431-preview.  Maybe you should try it.

Reply:
And it works..
![WindowsSandboxClient_xVKY6yDkmr](https://user-images.githubusercontent.com/15158490/155525271-a14af4d0-8e6e-4b57-9fba-64023f35508b.png)

Reply:
![Untitled](https://user-images.githubusercontent.com/7548515/155540148-4a1f6b39-7217-49dc-8fb0-42ef20300b17.jpg)

So strange. Why?

Reply:
You need to have a older version installed to use upgrade.

Reply:
see my screenshot. I have v6.21.22 installed

Reply:
All versions of `evernote.yinxiang` have a different `ProductCode` because the Installer unpacks a MSI installer which changes `ProductCode` for each MSI Installer creation.

I cheated and changed the `DisplayVersion` from `7.0.6.4762` to `7.0.6.4761`.

The only thing a `ProductCode` does is link that specific `ProductCode` to the specific `PackageIdentifier`, it has nothing to do with upgrading, the Previous versions of `Yinxiang Biji` is simply not detected by winget because it cannot match the `DisplayName` to `Yinxiang Biji v. 7.0.6` BECAUSE of the `v. `

The PR will not be accepted without the `AppsAndFeaturesEntries` fields.
Once this is merged you can fix the other versions by adding the `ProductCode` and `AppsAndFeaturesEntries` if you want.

Reply:
So I had know why I always failed to upgrade this, because what I try is v6 to v7 upgrade. That's what I argue for.
AppsAndFeaturesEntries can fix the problam of v7 to v7 upgrade, for this special package, but not v6 to v7 upgrade.

FYI, The DisplayName in this package is locale related. It's 印象笔记 v. 7.0.6 when installed in Chinese Windows System.
winget seems didn't check this at this moment, because adding InstallerType: msi in AppsAndFeaturesEntries is enough for upgrade this package. But It's always better to provide more information as much as possible.

Reply:
> FYI, The DisplayName in this package is locale related. It's 印象笔记 v. 7.0.6 when installed in Chinese Windows System.
winget seems didn't check this at this moment

Aha yes that is another point of failure, since you're added the Chinese locale it will use this metadata for future matching and maybe fix the current matching IF the DisplayName and Publisher is correct according to the registry/`A`dd or `R`emove `P`rograms